src/WebJobs.Script/Content/Script/functions.js

```diff
@@ -49,7 +51,15 @@ function createFunction(f) {
         });
         context.log = log;
 
+        if (typeof func === "string") {
+            systemLog("Requiring user function script.");
+            func = require(func);
+        }
+
+        func = getEntryPoint(func, context);
+
         context.done = function (err, result) {
+            systemLog("context.done called.");
             if (context._done) {
                 if (context._promise) {
                     context.log("Error: Choose either to return a promise or call 'done'.  Do not use both in your script.");
```

Comment:
Totally confused on this...

How do I call context.done from within a callback?


eg:
```
 request.post(url, {json: results}, (error, response, body) => {

            console.log(response.statusCode);

            if (response.statusCode == 500) {
                ctx.log.error(error);
                ctx.log.error(response);
            }


            ctx.done(null, JSON.stringify(results, null, 2));
            
        });
```